go.mod

```diff
@@ -4,19 +4,20 @@ go 1.18
 
 require (
 	github.com/Masterminds/semver/v3 v3.1.1
+	github.com/NVIDIA/k8s-operator-libs v0.0.0-20230118182127-8e73980389ca
 	github.com/caarlos0/env/v6 v6.4.0
 	github.com/go-logr/logr v1.2.3
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.3.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.20.1
+	github.com/onsi/gomega v1.24.0
 	github.com/openshift/api v0.0.0-20210428205234-a8389931bee7
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.8.0
+	github.com/stretchr/testify v1.8.1
 	k8s.io/api v0.25.5
 	k8s.io/apimachinery v0.25.5
 	k8s.io/client-go v0.25.5
 	k8s.io/kubectl v0.25.5
```

Comment:
It look like kubectl dependency is not required anymore. Could you please do `go mod tidy`